__tests__/__resources__/properties/default_properties.yaml

```diff
@@ -2,43 +2,43 @@
-#-- please also update the ITestPropertiesSchema interface so that          --#
-#-- developers can use auto-complete when using the properties in test code --#
+#------------------------------------------------------------------------------------#
+#  find and replace the following:                                                   #
+#                                                                                    # 
+#  zosmf-user               - This is your TSO user ID to use with z/OSMF            #
+#  zosmf-password           - This is your TSO password to use with z/OSMF           #
+#  zosmf-host               - The host name for the LPAR with z/OSMF                 #
+#  zosmf-port               - The z/OSMF port                                        #
+#  tso-account              - This is your JES Account number for batch jobs         #
+#  unix-testdir             - A directory on USS in which files and                  #
+#                             directories can be created                             #
+#  zosjobs-sysaff           - System Affinity for some batch jobs, use the system    #
+#                             that you are running the tests on                      #
+#  zosjobs-iefbr14Member    - PDS qualified by an IEFBR14 member JCL to submit       #
+#  zosjobs-iefbr14PSDataSet - PS with an IEFBR14 JCL to submit                       #
+#  console-sysplexsystem    - system name of a system in your sysplex to which       #
+#                             console commands can be routed                         #
+#  datasets-vol             - volume serial used to create data sets                 #
+#  ssh-host                 - host to connect to ssh on z/OS                         #
+#  ssh-port                 - port to connect to ssh on z/OS - default is 22         #
+#  ssh-privateKey           - private key for SSH. specify null if no key            #
+#  ssh-keyPassphrase        - passphrase for private key. specify null if            #
+#                             none                                                   #
+#  ssh-handshakeTimeout     - timeout for SSH connection. Specify null to use        #
+#                             the default or 0 for no timeout                        #
+#------------------------------------------------------------------------------------#
+#----- Note: If you add or rename any fields of this default template,          -----#
+#----- please also update the ITestPropertiesSchema interface so that           -----#
+#----- developers can use auto-complete when using the properties in test code  -----#
 
 #-------------------------------------------------------------------------#
 #  Connection details/credentials for z/OSMF                              #
 #-------------------------------------------------------------------------#
 zosmf:
   # your z/OSMF User ID
-  user: &zosmfuser my-user-name
+  user: &zosmfuser zosmf-user
```

Comment:
Thanks for updating.
Question, should the `&reference` be removed since they are not used in the yaml file?

Reply:
I think they exist for convenience so that the same username and password can be used in the SSH credentials below.

Reply:
So, should we use it (i.e. `*zosmfuser`) and remove the `ssh-user` (and password too)?